### PR TITLE
TEST: Check if GitHub Pages works well with Redux library

### PR DESCRIPTION
**Pull request by elzbthbrst**
I want to test how Redux work with github-pages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -39,7 +44,7 @@
   },
   "devDependencies": {
     "@babel/plugin-proposal-private-property-in-object": "^7.21.11",
-    "sass": "^1.64.1",
-    "gh-pages": "^5.0.0"
+    "gh-pages": "^5.0.0",
+    "sass": "^1.64.1"
```
Comment (babaruh) [inline package.json:48]: what is the impact of changing the order?

File: src/components/chatWindow/messages/AnotherMessage.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+                    <span className='time-mes'>12:28</span>
+                </div>
+                <div className="another__kebab">
+                    {isOpen && id === message.id && <MesAnotherKebab message={message}/>}
+                    <img className="cursor-pointer " src={menu_kebab} alt="" onMouseDown={onKebabClick} />
+                </div>
+            </div>
+        </div>
+    )
+}
```
Comment (babaruh) [inline src/components/chatWindow/messages/AnotherMessage.jsx:28]: https://github.com/kpdchat/kpdchat-backend/pull/21/#discussion_r1289164435

File: src/components/chatWindow/messages/SelfMessage.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import React from "react"
+import user_photo from '../../../images/chat-window/user-photo.png'
+import menu_kebab from "../../../images/chat-window/menu-kebab.png"
+import MesSelfKebab from "./mes-kebab/MesSelfKebab"
```
Comment (babaruh) [inline src/components/chatWindow/messages/SelfMessage.jsx:4]: is Mes a Message? if so, consider renaming all Mes to Message for better readability

File: src/components/chatWindow/messages/mes-kebab/MesAnotherKebab.jsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+    function onCopyClick() {
+        setCopy(true)
+        navigator.clipboard.writeText(message.text)
+    }
+    return (
+        <KebabWrapper elRef={menuRef} >
+
+            <div ref={menuRef} className="kebab-menu another-kebab">
+                <div className="another-kebab__row cursor-pointer" >
```
Comment (babaruh) [inline src/components/chatWindow/messages/mes-kebab/MesAnotherKebab.jsx:19]: consider renaming `another-kebab__row` to something readable

File: src/components/chatWindow/messages/mes-kebab/MesTitleKebab.jsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+    const menuRef = useRef()
+
+    function onOutClick() {
+        setOut(true)
+    }
+    
+    return (
+        <KebabWrapper elRef={menuRef}>
+            <div ref={menuRef} className="title-kebab kebab-menu">
+                <p  className="title-kebab__sound title-h4 cursor-pointer">Додати у папку</p>
```
Comment (babaruh) [inline src/components/chatWindow/messages/mes-kebab/MesTitleKebab.jsx:16]: remove double space like
`<p className="title-kebab__sound title-h4 cursor-pointer">Додати у папку</p>`

File: src/components/chatWindow/messages/mes-modal/ChatOutModal.jsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+        setOut((prev) => !prev)
+        dispatch(setKebabClose())
+    }
+    function onContentClick(e) {
+        e.stopPropagation()
+    }
+
+    return (
+        <div className="modal-container chat-out" onClick={onCloseClick}>
+            <div className="chat-out__content " onClick={onContentClick}>
```
Comment (babaruh) [inline src/components/chatWindow/messages/mes-modal/ChatOutModal.jsx:18]: useless space in `"chat-out__content "`. better to keep same code style

File: src/components/chatWindow/messages/mes-modal/ChatOutModal.jsx [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+                <div className="chat-out__message ">
```
Comment (babaruh) [inline src/components/chatWindow/messages/mes-modal/ChatOutModal.jsx:19]: same

File: src/components/chatWindow/messages/mes-modal/CopyModal.jsx [hunk 8]
```diff
@@ -0,0 +1,24 @@
...
+        setCopy((prev) => !prev)
+        dispatch(setKebabClose())
+    }
+    function onContentClick(e) {
+        e.stopPropagation()
+    }
+
+    return (
+        <div className="modal-container copy" onClick={onCloseClick}>
+            <div className="copy__content " onClick={onContentClick}>
```
Comment (babaruh) [inline src/components/chatWindow/messages/mes-modal/CopyModal.jsx:18]: same

File: src/components/chatWindow/navigation/ChatNavigation.jsx [hunk 9]
```diff
@@ -1,24 +1,24 @@
...
+import NavInfo from "./NavInfo";
 
 export default function ChatNavigation() {
+    
     return (
         <section className='chat__navigation navigation'>
             <div className='navigation__folders folders'>
                 <div className='folders__title'>
-                    <h2 className='title-h2'> не папки</h2>
+                    <h2 className='title-h2 '> папки</h2>
```
Comment (babaruh) [inline src/components/chatWindow/navigation/ChatNavigation.jsx:13]: same

File: src/styles/chat-window/dialogs.scss [hunk 10]
```diff
@@ -1,5 +1,12 @@
+@mixin max-screen($res) {
+    @media screen and (max-width: $res) {
+        @content;
+    }
+}
+
+
 .dialogs {
-    padding: 0 0 0 10px;
+    padding: 0 0 0 10px; //!!!!!!!!
```
Comment (babaruh) [inline src/styles/chat-window/dialogs.scss:9]: attention or what?

File: src/components/chatWindow/messages/mes-kebab/MesAnotherKebab.jsx (hunk 4 above)
Comment (elzbthbrst) [reply src/components/chatWindow/messages/mes-kebab/MesAnotherKebab.jsx:19]: i don`t have idea for naming this class, i have 2 types of messages : self-message and another-message, so if i have self-kebab, i should have another-kebab too

Comment (babaruh) [reply src/components/chatWindow/messages/mes-kebab/MesAnotherKebab.jsx:19]: okay

**Review by babaruh — APPROVED** [review]

**Pull request merged**